### PR TITLE
Local build

### LINKED ISSUE #510: Ontime window does not launch in local builds
When the app is packaged inside electron, it is electron's responsibility to start all the processes.
From electron's perspective, it goes something like

1. Open preload window
2. Start Ontime server (returns a port indicating where to find the server)
3. Close the preload window and open a window with the UI build served at the given port

It seems that the app gets stuck somewhere between step 2 and 3.
The server process is started by the preload window is never closed but the server is available

It is puzzling that the pipeline builds work well.
The only apparent difference is the inclusion of the local environment variable

This bug was likely introduced by the changes in #476 

We are likely calling the backend at the wrong place. A good place to start is the handling of the backend start call
https://github.com/cpvalente/ontime/blob/master/apps/electron/main.js#L159

### PR DESCRIPTION
**Pull request by alex-Arc**
fixes #510
the issue comes from build:local in apps/server builds' index and not app that is needed for electron dist

needs testing on mac and linux
- [x] Mac os
- [ ] Linux
- [x] Win

## Summary by CodeRabbit

- **Documentation**
	- Updated the build command instructions for UI and server in the development guide to reflect the new process for generating project distributions.

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ You can generate a distribution for your OS by running the following steps.
 From the project root, run the following commands
 
 - __Install the project dependencies__ by running `pnpm i`
-- __Build the UI and server__ by running `turbo build:local`
+- __Build the UI and server__ by running `turbo build:electron`
```
Comment (cpvalente) [inline DEVELOPMENT.md:79]: I am unsure on what I missed, but the `build:electron` scripts seem like copies of `build:local`

I am unsure on how anything is different here?

File: apps/server/package.json [hunk 2]
```diff
@@ -49,6 +49,7 @@
     "dev:test": "cross-env IS_TEST=true nodemon --exec \"ts-node-esm\" ./src/index.ts",
     "prebuild": "pnpm setdb",
     "build": "pnpm prebuild && esbuild src/app.ts --log-level=error --platform=node --format=cjs --bundle --minify --outfile=dist/index.cjs",
+    "build:electron": "pnpm prebuild && esbuild src/app.ts --log-level=error --platform=node --format=cjs --bundle --minify --outfile=dist/index.cjs",
```
Comment (alex-Arc) [inline apps/server/package.json:52]: this is the "big" difference
building `src/app.ts` and not `src/index.ts` 

I made a separate electron script to simplify the build to one turbo command but maybe there is a better way?

Comment (cpvalente) [reply apps/server/package.json:52]: oh I see, but then do we still need the `build:local` or is just a broken script? could the new script replace `build:local` instead?

Comment (alex-Arc) [reply apps/server/package.json:52]: if one wants to to run the apps/sever an apps/electron separate then  `local`  is the one that is needed

**Review by cpvalente — APPROVED** [review]
Good work

**Pull request merged**